go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/hashicorp/vault/api v1.0.2
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/influxdata/cron v0.0.0-20191203200038-ded12750aac6
-	github.com/influxdata/flux v0.91.0
+	github.com/influxdata/flux v0.91.1-0.20201029180714-1e9bfd49f21c
```

Comment:
I thought that we are only upgrading Flux in OSS to actual releases? @rockstar Can you confirm this?

Reply:
There was a bug in one of the flux window functions that the aggregate array cursors call. A few of the launcher test caught it. These changes depend on that fix.